src/Microsoft.AspNetCore.Mvc.Core/RequestSizeLimitAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Sets the <see cref="IHttpMaxRequestBodySizeFeature.MaxRequestBodySize"/> to the specified <see cref="Bytes"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class RequestSizeLimitAttribute : Attribute, IFilterFactory, IOrderedFilter
+    {
+        /// <summary>
+        /// The request body size limit.
+        /// </summary>
+        public long Bytes { get; set; }
```

Comment:
Did y'all discuss making this a constructor parameter vs property earlier? This seems odd to me as a property because `[RequestSizeLimit]` will set it to 0.